trk/src/Screens/TabScreens/Profile/Backend/ClimberProfile.js

```diff
@@ -40,7 +49,12 @@ const ClimberProfile = ({ navigation }) => {
     const handleTapHistory = async () => {
         const { getTapsBySomeField, getActiveSessionTaps, getRecentFiveSessions, getExpiredTaps, getTotalSessionCount} = TapsApi();
         const { getClimb } = ClimbsApi();
+        const {getUsersBySomeField} = UsersApi();
```

Comment:
Same reasoning as in `UserEdit.js`. Every time the `handleTapHistory` code block is called, it makes a call to the db (cf. `UsersApi.js`)

Reply:
The thing is in this case I'll need to instantiate these functions anyway. Maybe I should think of a check like if the number of climbs have been updated (using count, that is faster), and accordingly fetch data?